src/main/java/com/almostreliable/summoningrituals/altar/AltarBlock.java

```diff
@@ -38,14 +38,19 @@
 
 public class AltarBlock extends Block implements EntityBlock {
 
-    static final BooleanProperty ACTIVE = BooleanProperty.create(Constants.ACTIVE);
-    private static final DirectionProperty FACING = BlockStateProperties.HORIZONTAL_FACING;
-    private static final VoxelShape SHAPE = Stream.of(
+    public static final BooleanProperty ACTIVE = BooleanProperty.create(Constants.ACTIVE);
+    public static final DirectionProperty FACING = BlockStateProperties.HORIZONTAL_FACING;
+    public static final VoxelShape SHAPE = Stream.of(
         Block.box(3, 0, 3, 13, 2, 13),
         Block.box(5, 2, 5, 11, 9, 11),
         Block.box(2, 9, 2, 14, 13, 14)
     ).reduce((v1, v2) -> Shapes.join(v1, v2, BooleanOp.OR)).get();
 
+    public AltarBlock(Properties properties) {
+        super(properties);
+        registerDefaultState(defaultBlockState().setValue(FACING, Direction.NORTH).setValue(ACTIVE, false));
+    }
```

Comment:
If you need a constructor like that, you can get rid of the other one and pass the block properties from the `Setup` class instead.